src/pid/RevController.java

```diff
@@ -0,0 +1,93 @@
+package pid;
+
+import com.revrobotics.CANSparkMax;
+import sensors.counter.RevEncoder;
+
+import static pid.RevConstant.DEFAULT_ENCODER_UNITS_PER_ROTATION;
+import static pid.RevConstant.DEFAULT_SLOT_ID;
+
+public abstract class RevController extends AbstractController {
+
+    protected CANSparkMax sparkMax;
+    protected RevEncoder revEncoder;
+    protected int slotId;
+    protected double firstError;
+
+    public RevController(CANSparkMax canSparkMax,
```

Comment:
Why do we call this parameter `canSparkMax` on 2 constructors and `sparkMax` on the other ones?